detox/e2e/test/messaging/message_posting.e2e.js

```diff
@@ -26,4 +29,26 @@ describe('Messaging', () => {
 
         await expect(element(by.text(text))).toExist();
     });
+
+    it('MM-T3187 Unread channels sort at top', async () => {
+        // # Open channel drawer (with at least one unread channel)
+        await element(by.id('channel_drawer_button')).tap();
+        await expect(element(by.id('channels_list'))).toBeVisible();
+
+        // # Verify unread channel(s) display at top of channel list (with mentions first, if any), in alphabetical order, with title "Unreads"
+        await expect(element(by.text('UNREADS'))).toBeVisible();
+        await expect(element(by.id('channel_item_display_name')).atIndex(0)).toHaveText(newChannel.display_name);
```

Comment:
As suggested by Joseph, please add another channel so we can validate that the channel in unread section is really sorted.